### ISSUE TITLE
| ./csgoserver details | Empty output for "Distro Details" + "Performance" + "DiskUsage" + "Server Details"

### ISSUE THREAD

**Issue by LaurentDumont** [GameServerManagers/LinuxGSM#283]
Hi!

Small issue with that seems to be the latest version. With a CentOS 6.6 server, I'm getting nothing in the sections : "Distro Details" + "Performance" + "DiskUsage" + "Server Details" when using the "csgoserver details" command.

From debugging the script, I can see that the ".fn_details_distro + .fn_details_performance +  .fn_details_disk hidden files are created and contains the correct data.

Example: 
Distro:£CentOS release 6.6 (Final)
Arch:£x86_64
Kernel:£2.6.32-504.3.3.el6.x86_64
Hostname:£centos2.coldnorthadmin.com
tmux:£tmux 1.6
GLIBC:£2.12

I can see the control character "£" is there and that the "column -s £ -t .hidden_file" is being run. Unfortunately, it seems that that specific command outputs nothing. I have the same issue running the command directly from the command line.

Basically, it goes from fn_details -> function fn_sourcedetails() -> fn_details_distro (for the variables) and then fn_details_os||performance...

I cannot test (for now) on a Ubuntu/Debian server.

**Comment by TomatoTornado** [comment]
Right, testing it a bit more.

It's the "£" character which is making the column command fail. I'm slightly incredulous that character encoding can make something fail in 2015 but it seems to be the case here. Putty also has some issues displaying the character and I sometimes get fuzzy blotches where the character should be.

Changing it to "&" solves the issue.

**Comment by dgibbs64** [comment]
Thanks for this. i used £ as I thought it was unlikely to be used and cause an accidental tab. A character like & might cause issues. I want to try and find out if there can be multiple control characters go together e.g &%& . This would resolve the issue

**Comment by dgibbs64** [comment]
http://stackoverflow.com/questions/12133062/formatting-lists-with-the-column-command-in-nix

Got my reference

**Comment by dgibbs64** [comment]
This issue should now be resolved with the latest commit.
https://github.com/dgibbs64/linuxgameservers/commit/6c97fa522f12edfb958bd519ba5123014af41c64
